Xcode/xcconfigs/Project.xcconfig

```diff
@@ -70,6 +70,8 @@ SYSTEM_HEADER_SEARCH_PATHS                         = $(SRCROOT)/../vendor/SQLite
 
 COMBINE_HIDPI_IMAGES                               = YES    // Stop Xcode from complaining
 
+SDKROOT                                            = iphoneos       // necessary for catalyst
```

Comment:
Necessary for catalyst but is it a problem for others?

Reply:
I think, this should work, since we specify destination when building for macOS. Anyways, I will validate by running the build script and create release package.

Reply:
I worry that building inside of Xcode will become screwy if you add this

Reply:
I validated by creating the release package and then build successfully `My Mac` and  `My Mac (Mac Catalyst)` for `CBL_EE_Swift` from inside the Xcode. Do you think we should validate something else?